clients/auth/auth.go

```diff
@@ -0,0 +1,431 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+	"github.com/influxdata/influx-cli/v2/pkg/influxid"
+)
+
+type Client struct {
+	clients.CLI
+	api.AuthorizationsApi
+	api.UsersApi
+}
+
+const (
+	ReadAction  = "read"
+	WriteAction = "write"
+)
+
+type token struct {
+	ID          string   `json:"id"`
+	Description string   `json:"description"`
+	Token       string   `json:"token"`
+	Status      string   `json:"status"`
+	UserName    string   `json:"userName"`
+	UserID      string   `json:"userID"`
+	Permissions []string `json:"permissions"`
+}
+
+type printParams struct {
+	deleted bool
+	token   *token
+	tokens  []token
+}
+
+type CreateParams struct {
+	clients.OrgParams
+	User        string
+	Description string
+
+	WriteUserPermission bool
+	ReadUserPermission  bool
+
+	WriteBucketsPermission bool
+	ReadBucketsPermission  bool
+
+	WriteBucketPermissions []string
+	ReadBucketPermissions  []string
+
+	WriteTasksPermission bool
+	ReadTasksPermission  bool
+
+	WriteTelegrafsPermission bool
+	ReadTelegrafsPermission  bool
+
+	WriteOrganizationsPermission bool
+	ReadOrganizationsPermission  bool
+
+	WriteDashboardsPermission bool
+	ReadDashboardsPermission  bool
+
+	WriteCheckPermission bool
+	ReadCheckPermission  bool
+
+	WriteNotificationRulePermission bool
+	ReadNotificationRulePermission  bool
+
+	WriteNotificationEndpointPermission bool
+	ReadNotificationEndpointPermission  bool
+
+	WriteDBRPPermission bool
+	ReadDBRPPermission  bool
+}
+
+func (c Client) Create(ctx context.Context, params *CreateParams) error {
+
+	orgID := params.OrgID.String()
+	orgName := params.OrgName
```

Comment:
Commented below, but I think we need to look up the org ID via the name if the ID isn't set

Reply:
Doing this would mean embedding OrganizationsApi into client for auth, do you think this is worth doing for this?

Reply:
Yes, creating via an org name isn't actually supported (I think the swagger API is being lazy and reusing the GET response for the POST body)

Reply:
Okay, I will make this change now, otherwise this is ready for review again